frontend/pages/waysToEarnRoute/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { i18n, useTheme } from '@shopgate/engage/core';
+import { I18n, LoadingIndicator, Route } from '@shopgate/engage/components';
+import SmilePanel from '../../components/SmilePanel';
+import SmileFooter from '../../components/SmileFooter';
+import config from '../../config';
+import { WAYS_TO_EARN_ROUTE } from '../../constants';
+import styles from './style';
+import connect from './connector';
+
+/**
+ * @returns {JSX}
+ */
+const WaysToEarnRoute = ({
+  points,
+  customerIsFetching,
+  options,
+  optionsIsFetching,
+}) => {
+  if (customerIsFetching || optionsIsFetching) {
+    return <LoadingIndicator />;
+  }
+
+  const { waysToEarn, colorConfig, loginPageText } = config;
+  const { View, AppBar } = useTheme();
+  const title = points
+    ? `${points} ${i18n.text('smile.points')}`
+    : loginPageText.headerText.primaryText;
+
+  const body = options.length ? (
+    <SmilePanel header={waysToEarn.header} options={options} location={WAYS_TO_EARN_ROUTE} />
+  ) :
+    (
+      <div className={styles.panelContainer}>
+        <I18n.Text string="smile.smile_error" />
+      </div>
+    );
+
+  return (
+    <View>
+      <AppBar
+        title={title}
+        backgroundColor={colorConfig.headerBackground}
+        textColor={colorConfig.headerFontColor}
```

Comment:
should add right attribute with value of null to prevent search icon

Reply:
Updated